src/helpers.js

```diff
@@ -38,6 +38,21 @@ export function extractRequestFiles(request) {
         return
       }
 
+      if (typeof Blob !== 'undefined' && node[key] instanceof Blob) {
```

Comment:
Actually, we don't need this code in this if body at all because it's exactly the same as in the if-statement above. You just need to modify the if-statement above to run for both File-s and Blob-s